web/src/pages/ShotAnalyzer/components/ShotChart.css

```
@@ -37,50 +39,1731 @@
+  }
+
+  .analyzer-mobile-stack-layer--analysis {
+    z-index: 3;
+  }
+}
+
+@media (max-width: 640px) {
+  .shot-analyzer-page {
+    --analyzer-mobile-analysis-gap: 1rem;
+    --shot-chart-mobile-card-height: 90vh;
+    --shot-chart-mobile-card-height: 90dvh;
```

Stylelint (17.13.0)

[error] Duplicate custom property "--shot-chart-mobile-card-height" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)